plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/grouped-product-selector/edit.tsx

```diff
@@ -19,21 +24,49 @@ export default function AddToCartWithOptionsGroupedProductSelectorEdit(
 	props: BlockEditProps< Attributes >
 ) {
 	const { className } = props.attributes;
-	const { current: currentProductType } = useProductTypeSelector();
 	const { product } = useProductDataContext();
-	const productType =
-		product.id === 0 ? currentProductType?.slug : product.type;
-
```

Comment:
`currentProductType` was only being used when `product.id` was `0`, which is the same logic that's used to render the fallback context. I believe `currentProductType` will always be `simple` when no product context is provided, so I decided to remove it.